spec/AutomatedLinkAcceptance+services.yaml

```diff
@@ -1,87 +1,99 @@
 servers:
-## TODO: Search and replace all "xx" in the uuid by the official abbreviation of the ApplicationName
+
 
```

Comment:
uuids:
the own application ID needs to be in lowercase letters in all uuids. i.e. instead of **ALA**-1-0-0 it needs to be **ala**-1-0-0